OutOfSchool/OutOfSchool.EmailSender/EmailSender.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System.Threading.Tasks;
+using MailKit.Net.Smtp;
+using Microsoft.Extensions.Options;
+using MimeKit;
+using MimeKit.Text;
+
+namespace OutOfSchool.EmailSender
+{
+    public class EmailSender : IEmailSender
+    {
+        IOptions<EmailOptions> _emailOptions;
+        IOptions<SmtpOptions> _smtpOptions;
+
+        public EmailSender(IOptions<EmailOptions> emailOptions, IOptions<SmtpOptions> smtpOptions)
+        {
+            _emailOptions = emailOptions;
+            _smtpOptions = smtpOptions;
+        }
+
+        public Task SendAsync(Message message)
+        {
```

Comment:
currently this method is used only in SendAsync(string email, string subject, string htmlMessage)
it lokks like in can be removed from the IEmailSender interface and converted to private method

Reply:
removed from interface and made it private